### PR TITLE
Feature/test suite

### PR DESCRIPTION
**Pull request by chipbell4**
The big merge!

I want to get this into master so we can begin adding features to to SpringRollConnect with the full test suite running

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -58,8 +59,31 @@ module.exports = function(grunt)
...
+      //     'src/**/*.js',
+      //     '!src/libs/*.js'
+      //   ]
+      // },
+      // app: {
+      //   src: [
+      //     'app/**/*.js',
+      //     '!app/public/**'
+      //   ]
+      // }
```
Comment (DylanOrsh) [inline Gruntfile.js]: Do we mean for this commented out code to still be here?

File: test/pages/game-slug.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+//   it(`I can't ${EDIT}`, () => {});
+//   it(`I can't ${ACCESS_PRIVILEGES}`, () => {});
+// });
+
+describe(`${PAGE} as a edit capable user`, () => {
+  beforeEach(async () => await init(1, 1, 'dev'));
+  it(`I can ${VIEW}`, viewTest);
+  it(`I can ${VIEW_RELEASES}`, viewReleasesTest);
+  it(`I can ${EDIT}`, editGameTest);
+  //TODO: Fix permissions to allow for edit capable users to view
```
Comment (SmithAF) [inline test/pages/game-slug.js]: Do we want to merge in #68 before merging to master?

File: package.json [hunk 3]
```diff
@@ -42,6 +59,9 @@
     "uuid": "^3.2.1"
   },
   "scripts": {
-    "test": "grunt default --verbose"
+    "test": "concurrently 'node server.js' 'npm run test:init' -k --prefix-colors 'yellow,green' --success first",
+    "test:loop": "concurrently 'node server.js' 'npm run test:infinite'  -k --prefix-colors 'yellow,green' --success first",
+    "test:init": "mocha --require @babel/register --timeout 10000 test/index.js",
+    "test:infinite": "while npm run test:init; do :; done"
```
Comment (DylanOrsh) [inline package.json]: Are these different test commands documented somewhere?

Comment (SmithAF) [reply package.json]: they are not, but you really just have to use `test`, the other commands are just for making the instructions more modular, or in the case of `test:loop`, run `test` it until it fails.

File: test/pages/game-slug.js (hunk 2 above)
Comment (chipbell4) [reply test/pages/game-slug.js]: yes

File: Gruntfile.js (hunk 1 above)
Comment (chipbell4) [reply Gruntfile.js]: Yes. @andrewstart [caught this earlier](https://github.com/SpringRoll/SpringRollConnect/pull/66#discussion_r258167100) :)

**Pull request merged**